examples/json/README.md

```diff
@@ -1,11 +1,8 @@
-# GreeterJson
+# Json
```

Comment:
I think it would be more consistent to have examples/json/Greeter - not the single json example directly in json.

All the other top-level dir for examples are in Pascal case.

Reply:
I agree with @bernardnormier, its more consistent with the other IDLs. Plus in the future we probably will want to have another JSON example.